### PR TITLE
support for sections in UITableView

### PR DESCRIPTION
**Pull request by yasirmturk**
making it possible to use sections, section headers etc

### CODE REVIEW

**Comment by ashfurrow** [comment]
Hey! Could you summarize the changes? I don't quite follow them...

**Comment by yasirmturk** [comment]
u can not use more than 1 sections in the UITableView with existing solution because UICollectionView.tag can only tell the row index

**Comment by ashfurrow** [comment]
Ah, gotcha. Makes sense, thanks!
